<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelGridDialog</class>
 <widget class="QDialog" name="VoxelGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voxel Grid Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/Registar.qrc">
    <normaloff>:/images/bunny.png</normaloff>:/images/bunny.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="leafSizeXLabel">
         <property name="text">
          <string>LeafSizeX</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leafSizeXLineEdit">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="leafSizeYLabel">
         <property name="text">
          <string>LeafSizeY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leafSizeYLineEdit">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="leafSizeZLabel">
         <property name="text">
          <string>LeafSizeZ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leafSizeZLineEdit">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="overwriteCheckBox">
         <property name="text">
          <string>overwrite</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>78</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>VoxelGridDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
